template2.html

```diff
@@ -0,0 +1,180 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title> Taher Haggui </title>
+        <link rel="stylesheet" type="text/css" href="styles/template2.css">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+    </head>
+    <body>
+        <section class="p">
+          <div>
+                <img  class="project_image" src="statics/images/Snapshoot_Portfolio.png" alt="snapshoot image project">
+                <h1>Tonic</h1>
+                <div id="general_inf">
+                    <p class="title">CANOPY</p>
+                    <img class="counter" src="statics/images/Counter.png" alt="counter"/>
+                    <p class="job-title">Back End Dev</p>
+                    <img class="counter" src="statics/images/Counter.png" alt="counter">
+                    <p class="year">2015</p>
+                </div>
+            
+               <p class="project_description">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+               </p>
+          </div>
+          <footer>
+             <div id="skills_0" class="skills">
+               <p>html</p>
+               <p>css</p>
+               <p>javaScript</p>
+             </div>
```

Comment:
- [ ] The list of technologies should be using `ul` tag with `li` tags inside, please consider changing it. (ul stands for an unordered list and li stands for list item). This way your code will be more semantic.👍🏻